bin/system-integrity

```diff
@@ -0,0 +1,263 @@
+
+
+class DpkgIntegrityIssue(object):
+    """
+    A wrapper for output from the `dpkg --verify --verify-format rpm` command.
+    Used in conjunction with the DpkgIntegrityChecker.
+    """
+
+    TYPE_ID = 'integrityissue'
```

Comment:
Can we briefly explain what this means?

Reply:
This is a simple ID that could be used by a factory to provide the right object initialised with previously serialised data. It's also used in the `initialise` to know whether the data belongs to the object. This will be useful if we start adding more types of issues (see TODO).